campaignion_tracking/package/src/main.js

```diff
@@ -11,13 +12,14 @@ import * as listener from './listener'
 var debug = parseInt(sessionStorage.getItem('campaignion_debug')) ? true : false
 
 // some out-of-the-box prefixes
-export const codePrefixes = []
+export const codePrefixes = ['gtm']
```

Comment:
Why are there tracker specific fragment prefixes?

Reply:
The reason was to be easily integratable with the facebook pixel code which is working that way.